travis.sh

```diff
@@ -3,11 +3,18 @@ set -e #break script on non-zero exitcode from any command
 gem install bundler
 bundle install
 
+case $OSTYPE in darwin*)
+    brew update > /dev/null
+    brew upgrade cmake
```

Comment:
I don't remember to be honest. It is also possible that I needed greater version of cmake for something else and it was mixed up in here.